Source/Scene/SkyAtmosphere.js

```diff
@@ -133,6 +139,13 @@ define([
     /**
      * @private
      */
+    SkyAtmosphere.prototype.setDayNight = function(enableLighting) {
```

Comment:
This could also use a better name.  I don't know what "DayNight" is.  Is it day?  Is it night?